include/rangers-api/Application/Player/GOCPlayerHsm.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+namespace app::player {
+    class GOCPlayerHsm : public hh::game::GOComponent {
```

Comment:
Starting braces should be started on a line to keep consistent with the original repository.